### DISCUSSION TITLE
Update PostgreSQL setup instructions for new contributers

### DISCUSSION THREAD

**Discussion by webyneter** [djangopackages/djangopackages#661] [Docs]
I've noticed we refer to the outdated version of PostgreSQL (judging by the one from `./compose/postgres/Dockerfile`) [in the docs](https://djangopackages.readthedocs.io/en/latest/postgresql_contributor_instructions.html#ubuntu).

**Comment by ryancheley** [comment]
The docs have been refactored and restructured so that this no longer appears to be an issue
